From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sat, 3 Oct 2015 23:56:20 +0200
Subject: Use debian/changelog timestamp for the build time of the manuals

Otherwise the manuals become arch-dependent, breaking co-installability of
the Multi-Arch: same development packages.
---
 man/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index c18cf5c..fe14e74 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -166,12 +166,11 @@ HTML_DOCS 		= $(dist_man_MANS:%=%.html) $(man_MANS:%=%.html)
 %.3: %.3.in $(autogen_common)
 	@echo Generating $@ man page && \
 	rm -f $@-t-t $@-t $@ && \
-	date="$$(LC_ALL=C date "+%F")" && \
 	awk "{print}(\$$1 ~ /@COMMONIPCERRORS@/){exit 0}" ${top_srcdir}/man/$@.in > $@-t-t && \
 	cat ${top_srcdir}/man/$(autogen_common) >> $@-t-t && \
 	awk -v p=0 "(\$$1 ~ /@COMMONIPCERRORS@/){p = 1} {if(p==1)print}" ${top_srcdir}/man/$@.in >> $@-t-t && \
 	cat $@-t-t | \
-		sed -e 's#@BUILDDATE@#'$$date'#g' \
+		sed -e 's#@BUILDDATE@#$(PACKAGING_DATE)#g' \
 		    -e 's#@COMMONIPCERRORS@##g' \
 		    > $@-t && \
 	rm -f $@-t-t && \
